src/Sentry/SentrySdk.cs

```diff
@@ -115,7 +185,7 @@ internal static IDisposable UseHub(IHub hub)
         [EditorBrowsable(EditorBrowsableState.Never)]
         public static void Close()
         {
-            var oldHub = Interlocked.Exchange(ref _hub, DisabledHub.Instance);
+            var oldHub = Interlocked.Exchange(ref _mainHud, DisabledHub.Instance);
```

Comment:
Now we're only disposing the main hub and not the other ones closed for each thread.

Reply:
when a thread is closed it calls the Dispose from the Hubs I suppose.
I could set _currentHub.Value = null but I'm unsure about the effect of it,